AnkiDroid/src/main/java/com/ichi2/anki/utils/AndroidUtils.kt

```diff
@@ -56,7 +57,11 @@ fun Context.openUrl(uri: Uri) {
     if (!AdaptionUtil.hasWebBrowser(this)) {
         val noBrowserMessage = getString(R.string.no_browser_msg, uri.toString())
         if (this is FragmentActivity) {
-            showSnackbar(noBrowserMessage)
+            showSnackbar(noBrowserMessage) {
+                setAction(android.R.string.copyUrl) {
+                    copyToClipboard(uri.toString(), failureMessageId = R.string.failed_to_copy)
+                }
```

Comment:
no need for `failureMessageId`, it's the default

```suggestion
                setAction(android.R.string.copyUrl) { copyToClipboard(uri.toString()) }
```

Reply:
Thanks for the info! I have committed it now.